### PR TITLE
js implementation to readme

### PR DESCRIPTION
**Pull request by drakeerv**

### CODE REVIEW

File: community/baseinput.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+		self.game.controls[axis] -= (-1 if mode % 2 == 0 else 1)
+		self.update_move(axis)
+
+	def start_modifier(self, mode):
+		if mode == self.ModifierMode.SPRINT:
+			self.game.player.target_speed = player.SPRINTING_SPEED
+
+	def end_modifier(self, mode):
+		if mode == self.ModifierMode.SPRINT:
+			self.game.player.target_speed = player.WALKING_SPEED
```
Comment (obiwac) [inline community/baseinput.py:125]: Missing newline.

File: community/audio/readme.txt [hunk 2]
```diff
@@ -1 +1 @@
-For music download .ogg or other from https://archive.org/details/C418-MinecraftSoundtrackVolumeAlpha/ and put them into the music dir
\ No newline at end of file
+For music download .ogg (or other) from https://archive.org/details/C418-MinecraftSoundtrackVolumeAlpha/ and put them into the music dir
```
Comment (obiwac) [inline community/audio/readme.txt:1]: Could you rename `readme.txt` -> `README.md`?

Also add backticks around `.ogg` and make the URL a proper.

**Review by obiwac — CHANGES_REQUESTED** [review]
Overall LGTM, could you separate your two changes into two separate PR's?

**Comment by drakeerv** [comment]
made this the readme add and I will make another pr for controller.py

**Pull request merged**